Cargo.toml

```diff
@@ -34,13 +34,16 @@ zerofrom = { version = "0.1.0", optional = true }
 [features]
 default = []
 
+chat = []
 emote = ["serde"]
+entitlements = []
 moderation = ["serde"]
 points = ["serde"]
 stream = ["serde"]
 timestamp = []
 user = ["serde"]
 goal = ["serde"]
+extensions = []
```

Comment:
should `entitlements` and `extensions` be plural?

Reply:
Granted everything else except `points` is singular, I think they should be singular as well ([diff](https://github.com/twitch-rs/twitch_types/compare/1034b4989520392ced280d769a6258b1c74ce128..4e805c66055f8d1bc1fc99ab5d0009799f9e39ee)).